frontend/src/metabase/query_builder/components/filters/modals/BulkFilterModal/BulkFilterModal.tsx

```diff
@@ -77,15 +94,17 @@ const BulkFilterModal = ({
     <ModalRoot>
       <ModalHeader>
         <ModalTitle>{getTitle(question, query)}</ModalTitle>
+        <FieldSearch value={searchQuery} onChange={setSearchQuery} />
         <ModalCloseButton onClick={onClose}>
           <Icon name="close" />
         </ModalCloseButton>
       </ModalHeader>
-      {sections.length === 1 ? (
+      {sections.length === 1 || searchItems ? (
```

Comment:
if the user is searching, we throw all the hits into a single section and hide the tabs